doc/cli.markdown

```diff
@@ -1443,8 +1443,12 @@ Sample registry-secrets YAML file:
 		username: '_json_key'
 		password: '{escaped contents of the GCR keyfile.json file}'
 
-If an option is not specified, and a secrets.yml or secrets.json file exists in
-the balena directory (usually $HOME/.balena), this file will be used instead.
+For an example using the Google Container Registry, check this page:
+https://gist.github.com/pdcastro/d9739b9b81ad75ab5957bd5b8b405080
```

Comment:
I really don't think we should be adding a gist to a production codebase - can you add this to the docs somewhere and then link there please?

Reply:
Agreed. As discussed in the CLI check-in, I've created a sample `balena-io-playground` project and replaced the URL.